protos/flyteidl/admin/version.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package flyteidl.admin;
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin";
+
+// Represents a request structure to get version of flyteadmin.
+message Version {
+    // Specifies the GIT sha of the build
+    string	Build = 1;
+
+    // Version for the build, should follow a semver
+    string Version = 2;
+
+    // Build timestamp
```

Comment:
how about google.protobuf.Timestamp as the type here?

Reply:
I am not sure. I tried `google.protobuf.empty` and my proto gen(pbjs) failed. I can try but not sure https://github.com/flyteorg/flyteidl/pull/106/files#diff-9324a86791de0a6631e4fd9acccbe51c4b5f06c33180fd732c69de560e363246R493